app/__init__.py

```diff
@@ -48,12 +48,48 @@ def create_app(config_class, database_uri=None):
         ]
     )
 
-    # Set content security policy
+    SELF = "'self'"
+
     csp = {
-        "default-src": f"'self' {app.config['FLASKS3_CDN_DOMAIN']}",
-        "script-src": f"'self' {app.config['FLASKS3_CDN_DOMAIN']}"
-        + " 'sha256-GUQ5ad8JK5KmEWmROf3LZd9ge94daqNvd8xy9YS1iDw='"  # pragma: allowlist secret
-        + " 'sha256-l1eTVSK8DTnK8+yloud7wZUqFrI0atVo6VlC6PJvYaQ='",  # pragma: allowlist secret
+        "default-src": f"'self' {app.config['FLASKS3_CDN_DOMAIN']} https://cdn.jsdelivr.net 'unsafe-inline' ",
+        "script-src": (
+            [
+                SELF,
+                "'unsafe-eval'",
```

Comment:
if there is no options around using these it would be good to figure out which ones we really need and which unsafe flags we can get rid of, also if we allow unsafe eval/inline scripts we should maybe look into sanitising any query strings we use - this could maybe be in a different ticket

Reply:
Removed unsafe eval and limited script-src & style-src to specific universal viewer sources